html/modules/custom/docstore/src/MetadataTrait.php

```diff
@@ -222,6 +222,7 @@ protected function mapOrCreateTerms($field_name, $values, $type, $provider, $ent
       // Create term.
       $params = [
         'label' => $value,
+        'author' => $provider->uuid(),
```

Comment:
author should be passed in, is different from provider

Reply:
is already passed to `buildItemDataFromMetaData`

Reply:
That's what I suspected, but I couldn't see where that's done - e.g. in a lookup in the AR script.

Reply:
buildItemDataFromMetaData gets the author from the params, so pass it to mapOrCreateTerms and it should be fine